client/a8c-for-agencies/sections/referrals/common/step-section-item/style.scss

```
@@ -121,3 +121,14 @@
 
 	}
 }
+
+.step-section-item__step-number {
+	color: var(--color-accent);
+	font-size: rem(16px);
+	font-weight: 600;
+	text-align: center;
+	width: 24px;
```

```suggestion
	width: 20px;
```

Since the border takes an additional 2px (20 + 2 + 2 = 24)